ruby_event_store/lib/ruby_event_store/broker.rb

```diff
@@ -2,6 +2,8 @@
 
 module RubyEventStore
   class Broker
+    attr_reader :subscriptions
```

Comment:
I think this exposes too much of subscriptions internals. The subscriptions object has methods to add & remove subscriptions & I would live to avoid manipulating it outside ot RES.

Maybe RES should publish some "system" events when subscription is added and removed? Than you could build the read model with list of current subscriptions or even have more sophisticated logic based on it? WDYT?

Reply:
We just make `attr_reader` public so we still can't make changes in subscriptions outside RES.
I don't see any problem here.

Reply:
But `attr_reader :subscriptions` exposes `Subscription` class instance, with methods to manipulate (add & remove) subscriptions. You can read only the object itself but you are still allowed to run any method on it.